### PR TITLE
Feature/new chimp

### PR DESCRIPTION
**Pull request by Leeaandrob**
Finish the configurations of chimp only.

### CODE REVIEW

**Comment by bent0b0x** [comment]
Still karma changes in here 😄

**Comment by Leeaandrob** [comment]
I removed the .circle/karma.circleci.conf.js @bent0b0x

**Comment by bent0b0x** [comment]
there is still a file rename in here related to `karma`. Check the [diff](https://github.com/Paratii-Video/paratii-portal/pull/330/files)

**Comment by jellegerbrandy** [comment]
We must do these changes both for karma and for chimp, same pattern, we can do this in a single PR. Or am I missing a point?

**Comment by bent0b0x** [comment]
I suggested just doing karma in another PR (as long as @Leeaandrob creates an issue for it) just because karma's is _slightly_ more complex. But we can do it in the same PR.

@Leeaandrob basically you need to take the objects that are passed to the karma functions and abstract those into their own modules that you can override and extend

**Comment by Leeaandrob** [comment]
@bent0b0x I will follow the @jellegerbrandy desire using the same PR.

File: .circleci/karma.circleci.conf.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const shared = require('../karma.conf.js')
+
+module.exports = function(karma) {
+  shared(karma)
```
Comment (bent0b0x) [inline .circleci/karma.circleci.conf.js]: I'm not sure extending it this way will work. Is this still work in progress?

Comment (Leeaandrob) [reply .circleci/karma.circleci.conf.js]: https://circleci.com/gh/Paratii-Video/paratii-portal/1291 @bent0b0x

**Pull request merged**